plugins/SegmentEditor/API.php

```diff
@@ -348,6 +349,36 @@ public function add(
         return $id;
     }
 
+    /**
+     * Stars a stored segment.
+     *
+     * @param int $idSegment
+     * @throws Exception if the user is not logged in or does not have the required permissions.
+     */
+    public function star(int $idSegment): void
+    {
+        $segment = $this->getSegmentOrFail($idSegment);
+        $this->checkUserCanEditOrDeleteSegment($segment);
+        $bind = array('starred' => 1);
```

Comment:
shorthand array syntax is preferred for new code

```suggestion
        $bind = ['starred' => 1];
```

Reply:
✅ Done!